### PR TITLE
Cache scraped data

### LINKED ISSUE #365: Cache scraped data + Add to Travis Build Cache
This would speed up tests and reduce the number of timeouts.
In particular, the following modules: `clock`, `ethnologue`, `iso639`
**Note:** Cache only the raw data downloaded, not the database generated from it.

### PR DESCRIPTION
**Pull request by Androbin**
Fixes #365

### CODE REVIEW

**Comment by Androbin** [comment]
The raw data is cached on disk and the polished data is kept in memory.
Thus, I decided to no longer store the polished data on disk.
This makes the code less error-prone and increases coverage.

**Comment by Androbin** [comment]
Caching the raw data is about as efficient as caching the polished data (setup only).
The raw data can be cached much more safely, as it doesn't depend on further processing.
This is an especially useful feature for CI builds.

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,10 @@ language: python
 
 sudo: false
 
-cache: pip
+cache:
+  pip: true
+  directories:
+  - ~/.phenny/cache
```
Comment (sushain97) [inline .travis.yml]: indentation

File: tools.py [hunk 2]
```diff
@@ -69,6 +68,30 @@ def read_db(self, name, warn_after=None):
     except:
         raise GrumbleError()
 
+def db_path(self, name):
+    return dot_path(self.nick + '-' + self.config.host + '.' + name + '.db')
```
Comment (sushain97) [inline tools.py]: % s or format would be nice here

**Pull request merged**